# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/HyperTextAccessible.cpp
# Commit: c0a7fdc4bfca
# Full Hash: c0a7fdc4bfca0596ffecd347f0e248440e371989
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P5: Unify GetLevelInternal and GetDefaultLevel as Accessible::GetLevel. r=Jamie
#   
#   The nsAccUtils method was a "fast" one for calculating set sizes and
#   conceptual parents. Unified it with
#   LocalAccessible::GetLevelInternal in Accessible::GetLevel with an
# ==============================================================================

diff -r f6ce0e05c92e -r c0a7fdc4bfca accessible/generic/HyperTextAccessible.cpp
--- a/accessible/generic/HyperTextAccessible.cpp	Tue Jan 04 21:01:35 2022 +0000
+++ b/accessible/generic/HyperTextAccessible.cpp	Tue Jan 04 21:01:36 2022 +0000
@@ -1239,13 +1239,6 @@
   return attributes.forget();
 }
 
-int32_t HyperTextAccessible::GetLevelInternal() {
-  if (auto* heading = dom::HTMLHeadingElement::FromNode(mContent)) {
-    return heading->AccessibilityLevel();
-  }
-  return AccessibleWrap::GetLevelInternal();
-}
-
 void HyperTextAccessible::SetMathMLXMLRoles(AccAttributes* aAttributes) {
   // Add MathML xmlroles based on the position inside the parent.
   LocalAccessible* parent = LocalParent();